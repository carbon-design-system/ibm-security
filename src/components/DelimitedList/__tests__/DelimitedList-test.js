/**
 * @file Delimited list tests.
 * @copyright IBM Security 2019
 */

import { shallow } from 'enzyme';
import React from 'react';

import { DelimitedList } from '../../..';

import props from '../_mocks_';

describe('DelimitedList', () => {
  it('renders correctly', () => {
    expect(shallow(<DelimitedList items={props.items} />)).toMatchSnapshot();
  });
});
