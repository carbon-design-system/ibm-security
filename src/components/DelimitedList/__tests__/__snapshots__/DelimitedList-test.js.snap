// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DelimitedList renders correctly 1`] = `
<div
  className="security--delimited-list"
>
  <div
    className="security--delimited-list__value security--delimited-list__value--truncated"
  >
    Item 1, Item 2, Item 3, Item 4, Item 5, Item 6, Item 7, Item 8, Item 9, Item 10
  </div>
</div>
`;
